tools/block-generator/runner/run.go

```diff
@@ -153,20 +157,28 @@ func (r *Args) run(reportDirectory string) error {
 	// get next db round
 	var nextRound uint64
 	var err error
-	if r.ResetDB {
-		fmt.Printf("%sPostgreSQL resetting.\n", pad)
-		if err = util.EmptyDB(r.PostgresConnectionString); err != nil {
-			return fmt.Errorf("emptyDB err: %w", err)
-		}
-		nextRound = 0
-	} else {
-		nextRound, err = util.GetNextRound(r.PostgresConnectionString)
-		if err != nil && err == util.ErrorNotInitialized {
+	switch r.Template {
```

Comment:
Instead of switching on the template, I would consider wrapping it in a check for `r.PostgresConnectionString != ""`

Reply:
Currently, supplying the connection string is ignored in the case of the file exporter, and only asserted for the pg exporter. I'm not sure it's worth re-prioritizing the logic, but I'm open to further discussion.